hr_family/models/hr_employee.py

```diff
@@ -19,20 +19,27 @@
 #
 #
 
-from openerp import models, fields
+from odoo import models, fields
 
 
 class HrEmployee(models.Model):
     _inherit = 'hr.employee'
 
-    fam_spouse = fields.Char("Name")
-    fam_spouse_employer = fields.Char("Employer")
-    fam_spouse_tel = fields.Char("Telephone.")
+    fam_spouse = fields.Char(string="Name")
```

Comment:
@GillesTephaneMeyomesse Can you change the string to be more detailed as "Spouse's Name", "Spouses's Employer", "Spouse's Telephone", if you will want to export it as csv/excel the headers will not be quite explicit...